crates/wash/src/cli/dev.rs

```diff
@@ -125,9 +127,15 @@ impl CliCommand for DevCommand {
             debug!("no recommendations found for project tools");
         }
 
-        // TODO(#17): spinner
-        debug!("building component");
-        // Build component
+        let spinner = ProgressBar::new_spinner();
+        spinner.set_style(
+            ProgressStyle::default_spinner()
+                .template("{spinner:.green} {msg}")
+                .unwrap(),
```

Comment:
```suggestion
                .context("failed to create spinner")?,
```

Would prefer to safely bail in the case that this fails for some reason